packages/build/src/telemetry/main.js

```diff
@@ -70,9 +70,8 @@ const getPayload = function ({ status, commandsCount, netlifyConfig, durationNs,
 }
 
 const addSiteInfoData = function (payload, siteInfo = {}) {
```

Comment:
Should this function be inlined to `getPayload()`?

```js
const getPayload = function ({ status, commandsCount, netlifyConfig, durationNs, siteInfo = {} }) {
  const basePayload = {
    user_id: 'buildbot_user',
    event: 'build:ci_build_process_completed',
    timestamp: Date.now(),
    properties: {
      status,
      steps: commandsCount,
      buildVersion: version,
      nodeVersion: nodeVersion.replace('v', ''),
      osPlatform: OS_TYPES[platform],
      osName: osName(),
      siteId: siteInfo.id, 
    },
  }
  return addDuration(addConfigData(basePayload, netlifyConfig), durationNs)
}
```

Reply:
Good one, was on a deletion rampage and missed that 😄 addressed via https://github.com/netlify/build/pull/2458/commits/75233de3b5982d4452e9bd8491bd45571dcd8ff0